edx_event_bus_kafka/internal/consumer.py

```diff
@@ -146,11 +146,12 @@ def reset_offsets(consumer, partitions):
             if offset_timestamp is None:
                 return
 
-            # Get the offset from the epoch in seconds.
-            offset_timestamp_s = int(offset_timestamp.timestamp())
+            # Get the offset from the epoch (Kafka expects offsets in milliseconds for offsets_for_times, see
+            # https://kafka-python.readthedocs.io/en/master/apidoc/KafkaConsumer.html#kafka.KafkaConsumer.offsets_for_times)
```

Comment:
Might be misleading to link to the kafka-python library's docs here, since we're not using it.

Reply:
Good catch. I'll see if I can find it in the library we're actually using.

Reply:
As far as I can tell it's undocumented, both for confluent-kafka and the underlying librdkafka. I have a PR open to add a note about it in the latter.